webclient/dokument/urls.py

```diff
@@ -42,7 +42,7 @@
     ),
     path("radek-tabulky-odkaz",views.get_dokument_table_row, name="get_dokument_table_row"),
     path(
-        "akce-get-3d",
+        "mapa-3d",
```

Comment:
v kontextu https://github.com/ARUP-CAS/aiscr-webamcr/issues/301#issuecomment-1802682882 - zde myslím stačí změnit na `model/mapa-3d` a je to